core/database/foxx/tests/user_router.test.js

```diff
@@ -0,0 +1,71 @@
+"use strict"
+// NOTE: completion of tests requires successful run of user_fixture.js script
+
+// Integration test of API
+const { expect } = require("chai");
+const request = require("@arangodb/request");
+const { baseUrl } = module.context;
+const { db } = require("@arangodb");
+
+const usr_base_url = `${baseUrl}/usr`
+
+
+// NOTE: describe block strings are compared against test specification during test call, not file name
+describe("user_router: the Foxx microservice user_router token/set endpoint", () => {
```

Comment:
Full testing may be out of scope for this ticket, but this is a great suggestion nonetheless.

It may also be within scope and very helpful to both review error conditions introduced in this PR and test for those.